src/sm/networkmanager/networkmanager.cpp

```diff
@@ -41,6 +41,13 @@ Error NetworkManager::Init(StorageItf& storage, cni::CNIItf& cni, TrafficMonitor
     return ErrorEnum::eNone;
 }
 
+NetworkManager::~NetworkManager()
+{
+    LOG_DBG() << "Destroy network manager";
```

Comment:
Remove this log, no other modules have it.

Reply:
Done